docs/pages/[id].js

```diff
@@ -0,0 +1,77 @@
+/*
+pages/[id].js renders a dynamic gestalt docs page with a with a page that isn't defined .
+
+The getStaticPaths() will look at the files in the ./markdown folder and try to render a page if it exists, or returns a 404.
```

Comment:
Instead of a 404, which we don't handle gracefully, can we redirect to the home page for unknown paths?
![Screen Shot 2022-07-08 at 5 24 55 PM](https://user-images.githubusercontent.com/12059539/178084428-a486b820-2f39-4df8-b6b8-485e94c3ee65.png)

Reply:
I don't think we can do a redirect at the moment. `GetStaticProps` handles the return value for `fallback` and it seems like 404 might be the way that it's handled.

https://nextjs.org/docs/api-reference/data-fetching/get-static-paths#getstaticpaths-return-values

However, it maybe possible to modify our 404 handler to redirect to the homepage perhaps?